assets/training/finetune_acft_hf_nlp/src/validate_lora_weights/validate_lora_weights.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Validate LoRA weights."""
+
+from peft import PeftModel
+from transformers import AutoModelForCausalLM, AutoTokenizer
+from azureml.acft.common_components import get_logger_app, set_logging_parameters, LoggingLiterals
+from azureml.acft.contrib.hf.nlp.constants.constants import LOGS_TO_BE_FILTERED_IN_APPINSIGHTS
+from azureml.acft.common_components.utils.error_handling.exceptions import ACFTValidationException
+from azureml.acft.common_components.utils.error_handling.error_definitions import ACFTUserError
+from azureml._common._error_definition.azureml_error import AzureMLError  # type: ignore
+from azureml.acft.common_components.utils.error_handling.swallow_all_exceptions_decorator import (
+    swallow_all_exceptions,
+)
+from azureml.acft.contrib.hf import VERSION, PROJECT_NAME
+import argparse
+import torch
+
+COMPONENT_NAME = "ACFT-Validate_lora_weights"
+
+logger = get_logger_app(
+    "azureml.acft.contrib.hf.scripts.components.scripts.validate_lora_weights.validate_lora_weights"
+)
+
+GENERATION_CONFIG = {
+    "max_new_tokens": 256,
+    "do_sample": True,
+    "top_p": 0.7,
+    "temperature": 0.8
+}
+
+
+def validate_lora_weights(base_model_path: str, lora_weights_path: str, tokenizer_path: str, test_examples: list):
+    """Load model and make forward pass to validate lora weights."""
+    logger.info("Validating lora model weights")
+
+    tokenizer = AutoTokenizer.from_pretrained(tokenizer_path)
```

Comment:
Given that the config is in a different folder, I think its best to pass config information while loading tokenzier and base model